script/unicode/Unicode/FetchDatabase.lean

```diff
@@ -0,0 +1,46 @@
+/-
+Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jean-Baptiste Tristan
+-/
+
+open System IO FilePath Process FS Std
+
+def major : Nat := 15
+def minor : Nat := 1
+def update : Nat := 0
+
+def unicodeUrl : String :=
+  s!"https://www.unicode.org/Public/{major}.{minor}.{update}/ucd/"
+
+def unicodeDatasets : List String := ["UnicodeData.txt"]
+
+def download  (url : String) (file : FilePath) : IO Output := do
+  if (¬ (← file.pathExists)) then
+    output { cmd := "curl", args := #["-s", "-S", "-f", "-o", file.toString, "-L", url] }
+  else pure { exitCode := 0, stdout := "", stderr := "" }
+
+def writeUnicodeVersion : IO Unit := do
+  let workingDir : FilePath ← currentDir
+  let f : FilePath := join workingDir <| System.mkFilePath ["..", "..", "src", "Init", "Data", "Char", "UnicodeVersion.lean"]
+  let mut content := ""
```

Comment:
In fact you can write the entire thing as a single string using the `\` trick from the suggestion below.